hadoop-ozone/ozone-recon-codegen/pom.xml

```diff
@@ -21,11 +21,10 @@
     <version>0.5.0-SNAPSHOT</version>
   </parent>
   <modelVersion>4.0.0</modelVersion>
-  <artifactId>hadoop-ozone-recon-codegen</artifactId>
+  <artifactId>hadoop-ozone-reconcodegen</artifactId>
```

Comment:
Why this naming is changed?
Now it is different from the project directory structure?

Reply:
Thanks got info from PR description.